css-layout-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  justify-content: space-between;
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-third {
+  justify-content: space-between;
+  flex-basis: calc((100% / 3 ) * 2);
+}
+
+.column-one-forth {
+  flex-basis: calc(100% / 4);
+}
+
+.too-big {
+  max-width: 200px;
+}
```

Comment:
if you make all of your images 100% the width of their containing block, then you won't need this